### ISSUE TITLE
Spanish should have at least two variants (Spain and Latin American)

### ISSUE THREAD

**Issue by Temporalin** [drwhut/tabletop-club#286]
**Is your feature request related to a problem? Please describe.**
The Spanish language is usually divided in Spanish from Spain and Spanish from Latin America. While speakers from one place can understand the other variant, we usually prefer to use our own.

**Describe the solution you'd like**
Split the Spanish language in, at least, two languages.
- Spanish (Spain) = Español (España)
- Spanish (LatAm) = Español (latino)

**Describe alternatives you've considered**
Some softwares have variants per country (Mexico, Argentina...).

**Additional context**
From the blog Flags are not Languages:
![alt text](https://www.flagsarenotlanguages.com/blog/wp-content/uploads/2022/09/Screen-Shot-2022-09-27-at-07.54.47.png)

**Comment by Temporalin** [comment]
Sorry if it sounded like I asked for the translation. Enabling the variants in Weblate is the way to go, of course. Thanks!

**Comment by GrimPixel** [comment]
Yes, it is easy to do. Godot engine has Spanish, Spanish (Argentina), Spanish (Mexico).
